src/styles/utils/spacing.ts

```diff
@@ -11,6 +11,10 @@ export default {
         margin: 0,
     },
 
+    m1: {
```

Comment:
Do we _have_ to add this value for literally only a single new case of margins? I think we can come up with something better :)

Reply:
I think this might be useful in the future as well. We don’t have m1:4 defined anywhere, but this kind of spacing scale does appear sometimes in mockups. If we never add it to our spacing system, we’ll always end up hardcoding it directly in styled components. It seems quite reasonable to include it in the spacing scale. We also discussed this with @SzymczakJ  during his review.